lib/web_console/templates/console.js.erb

```diff
@@ -446,6 +450,24 @@ REPLConsole.installInto = function(id, options) {
 // It allows to operate the current session from the other scripts.
 REPLConsole.currentSession = null;
 
+REPLConsole.XMLHttpRequest = typeof XMLHttpRequest === 'undefined' ? null : XMLHttpRequest;
```

Comment:
This line is for the Firefox Add-on. It doesn't have XMLHttpRequest as default, and so it needs to require a compatible module (see also here: http://git.io/vYW6X).